azure-pipelines/publishing/github-release.yml

```diff
@@ -6,7 +6,7 @@ trigger:
 pr:
 - none
 
-pool: dotestlab
+pool: 1es_hosted_pool_ubuntu_1804
```

Comment:
This should actually be using the msft hosted agent, until 1ES machine can support docker